payments/schema/mutations.py

```diff
@@ -584,6 +586,70 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteOrderLineMutation()
 
 
+class SendBerthSwitchOfferMutation(graphene.ClientIDMutation):
+    class Input:
+        offers = graphene.List(graphene.NonNull(graphene.ID))
+        due_date = graphene.Date(description="Defaults to the Offer due date")
+        profile_token = graphene.String(
+            required=False, description="API token for Helsinki profile GraphQL API",
```

Comment:
Required is `False` by default
```suggestion
            description="API token for Helsinki profile GraphQL API",
```

Reply:
fixed